usr/share/lib/img_proof/tests/SLES/test_sles_hardened.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+
+
+# Workaround for https://github.com/OpenSCAP/openscap/issues/1796
+def setup_swap(host, swap_file):
+    mem_total = int(host.run(
+        "cat /proc/meminfo | awk '$0 ~ /MemTotal:/ { print $2 }'"
+    ).stdout.strip())
+    if mem_total < 16e6:
+        for command in [
+            "sudo fallocate -l 4G {}".format(swap_file),
+            "sudo chmod 600 {}".format(swap_file),
+            "sudo mkswap {}".format(swap_file),
+            "sudo swapon {}".format(swap_file),
+        ]:
+            result = host.run(command)
+            if result.rc != 0:
+                raise RuntimeError(result.stderr.strip())
```

Comment:
This changes behavior. It was a pytest skip and now leads to a pytest.fail. I think the pytest.skip call is fine here and the cleanup commands could be still referenced here but maybe moved to a separate function?

Reply:
Fixed with a context manager.